tests/inject/karma.conf.js

```diff
@@ -1,19 +1,40 @@
+// @ts-check
 const fs = require('fs');
 const os = require('os');
 const rollupPluginIstanbul = require('rollup-plugin-istanbul2');
 const rollupPluginNodeResolve = require('@rollup/plugin-node-resolve').default;
-const rollupPluginReplace = require('@rollup/plugin-replace');
-const rollupPluginTypescript = require('@rollup/plugin-typescript');
 const typescript = require('typescript');
 const {getTestDestDir, rootPath} = require('../../tasks/paths');
 
-module.exports = (config) => {
-    config.set({
-        basePath: '../../',
+/**
+ * @typedef {import('karma').Config} Config
+ * @typedef {import('karma').ConfigOptions} ConfigOptions
+ * @typedef {import('rollup').Plugin} RollupPlugin
+ * @typedef {import('@rollup/plugin-typescript').RollupTypescriptPluginOptions} RollupTypescriptPluginOptions
+ * @typedef {import('@rollup/plugin-replace').RollupReplaceOptions} RollupReplaceOptions
+ */
+
+/** @type {(options?: RollupReplaceOptions) => RollupPlugin} */
+// @ts-ignore
+const rollupPluginReplace = require('@rollup/plugin-replace');
+/** @type {(options?: RollupTypescriptPluginOptions) => RollupPlugin} */
+// @ts-ignore
+const rollupPluginTypescript = require('@rollup/plugin-typescript');
```

Comment:
These `@ts-ignore`s are because the typing was not working. It was saying that these modules' default export are not functions, but they really are. This covers up the problem and does not fix it.

Reply:
If you know a trick to fix this, I would be interested in hearing it. But the code works this way.

Reply:
Hmm we have other files that also imports these, and they import it correctly, seems it would be fixed with a little config trickery?

Reply:
Well I mean, this file was full of red marks when I added the `@ts-check`. But it's really useful to be able to see the documentation and completion that comes when you fully type something.